test/Microsoft.ML.Tests/Microsoft.ML.Tests.csproj

```diff
@@ -36,6 +36,6 @@
     <None Include="Scenarios\Api\AspirationalExamples.cs" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Microsft.ML.TensorFlow.TestModels" Version="0.0.2-test" />
+    <PackageReference Include="Microsoft.ML.TensorFlow.TestModels" Version="0.0.3-test" />
```

Comment:
>-test [](start = 81, length = 5)

just out of curiosity, are we going to remove it (`-test`) sometime or we actually named it like this?